README.md

```diff
@@ -1,44 +1,88 @@
 # Formal Ledger Specifications
 
-This repository contains the formal ledger specifications that are intended to eventually replace the existing formal specifications of the Cardano ledger found [here](https://github.com/IntersectMBO/cardano-ledger). This project is currently incomplete and work in progress.
+This repository contains the formal ledger specifications for the Cardano blockchain,
+written in [Agda](https://wiki.portal.chalmers.se/agda/pmwiki.php). These
+specifications are executable, and Haskell code can be extracted from the Agda source
+code and run for conformance testing.
 
-This repository currently contains the work-in-progress specification for Cardano (up to and including the Conway era). The specification is executable and contains some documentation in the form of a PDF document. It can be built by following the steps in [Building](#building).
+The project consists entirely of literate Agda code from which we generate documentation,
+viewable in a web browser, that replaces the TeX/PDF-based specifications of previous eras.
 
-| Formal Specification | HTML Version |
-| -------------------- | ------------ |
-| [Full Cardano Ledger](https://IntersectMBO.github.io/formal-ledger-specifications/cardano-ledger.pdf) | [Ledger](https://IntersectMBO.github.io/formal-ledger-specifications/html/index.html) |
-| [Conway](https://IntersectMBO.github.io/formal-ledger-specifications/conway-ledger.pdf) | [Ledger](https://IntersectMBO.github.io/formal-ledger-specifications/html/index.html) |
+The repository provides the specification for the Cardano ledger up to and including
+the Conway era.   The Agda formalization of the Conway era is complete, whereas that
+of previous eras is still work-in-progress.
```

Comment:
```suggestion
The repository provides the specification for the Cardano ledger up to and including
the Conway era. The Agda formalization of the Conway era is complete, whereas that
of previous eras is still work-in-progress.
```

Reply:
I've already rewritten this paragraph and it no longer has three white spaces after a period (though it does have two white spaces!)